grpc_connect_options.go

```diff
@@ -132,18 +132,28 @@ func UnaryClientInterceptor(opts *GRPCUnaryInterceptorOptions) grpc.UnaryClientI
 
 		if o.UseCircuitBreaker {
 			success := make(chan bool, 1)
+			ignoredError := make(chan error, 1)
 			errC := hystrix.GoC(ctx, method, func(ctx context.Context) error {
 				err := o.retryableInvoke(ctx, method, req, reply, cc, invoker, opts...)
-				if err == nil {
+				switch {
+				case err == nil:
 					success <- true
+					return nil
+				case status.Code(err) == codes.NotFound: // to prevent circuit breaker open by notfound error
+					ignoredError <- err
+					return nil
+				default:
+					return err
 				}
-				return err
 			}, nil)
 
 			select {
 			case out := <-success:
 				logrus.Debugf("success %v", out)
 				return nil
+			case err := <-ignoredError:
+				logrus.Warnf("failed %s", err)
```

Comment:
gaperlu diwarn

Reply:
iya error not found kita banyak soalnya. terlalu banyak nanti lognya